scheduler/src/cook/scheduler/constraints.clj

```diff
@@ -81,7 +81,8 @@
   (->> (:job/instance job)
        (remove #(true? (:instance/preempted? %)))
        (mapv :instance/hostname)
```

Comment:
I thought `mapv` is not lazy?

Reply:
`distinct` is lazy https://clojuredocs.org/clojure.core/distinct